sdk/monitor/monitor-opentelemetry-exporter/src/utils/metricUtils.ts

```diff
@@ -106,3 +134,22 @@ export function resourceMetricsToEnvelope(
 
   return envelopes;
 }
+
+export function isAksAttach(): boolean {
+  return !!(
+    process.env[ENV_AZURE_MONITOR_AUTO_ATTACH] === "true" && process.env.AKS_ARM_NAMESPACE_ID
```

Comment:
suggestion: these accesses look like candidates for helper function(s) -- but that could just be me leaning into code minification -- as that also requires and additional function call which is more expensive on CPU vs number of bytes downloaded / installed.